Gaijinizer patch/Scripts.txt

```diff
@@ -3100,33 +3099,33 @@ What you are doing 'intervention' should have something to do with it.
  我々は、その時を心待ちにしております。
  =================================================="
 > CONTEXT: Scripts/テキスト出力/212:12
-"Subject: Congratulations
-Sender [Meira]
-
- 《Text》==========================================
- Dr. Kurosawa. Has invented another means of communication than the 'device'
- We will send you a message here.
-
-First of all, congratulations on 'completion of the story'.
- Whatever the outcome, he escaped that city
- The 'help' you can do in the world is over.
-But...I think you feel thin
- Apparently, it seems far from our hope of ending.
-
-If you are not convinced by this end
- If you are prepared to aim for a better end ...
-We are for you to see that world again
-There is a preparation.
-Of course, it is not 'the same world.'
-You will need to choose 'Start from scratch'.
-
-But you are different from the first time ...
-By leading one story to the end
- 'The girls' in my room will surely help you.
-Your experience and mind corruption will be taken over and helpful.
-Please be courageous and step into that world again.
- We look forward to that time.
- =================================================="#MTLed
+"Sender [Meira]
+Subject: Congratulations
+
+ 《Body》==========================================
+As Professor Yui has invented a device that allows for another method
+of communication, I shall send you a message from here.
+
+First of all, congratulations on completing the story. Regardless
+of the outcome, he escaped that city, and the assistance you've
+rendered from your world is now over. However, I'm sure you'll
+be able to appreciate on some level that this does not appear to
+be the conclusion we had hoped for.
+
+Supposing you are not satisfied with this ending, and you're prepared
+to aim for a better conclusion, we are willing to provide another
+opportunity to peek into that world. It won't be the same world,
+of course. And you'll have to choose to start from the beginning.
+
```

Comment:
"Start from the beginning" here is a flat out reference to the menu command: 
![image](https://user-images.githubusercontent.com/48231102/92357746-93e8d080-f0a5-11ea-98de-e4b7e7760aef.png)

Knowing this, a few changes to the flow:

```
Supposing that you are not satisfied with this ending, and you're prepared
to aim for a better conclusion...

We are willing to provide you with another chance to peer into that world.
It won't be the 'same' world of course. You'll have to choose [Start From The
Beginning].
```